src/pages/api/v1/manufacturers/list.ts

```diff
@@ -0,0 +1,35 @@
+import { routes } from "@seamapi/types/devicedb"
+
+import { withRouteSpec } from "lib/middleware/index.ts"
+import { publicMapManufacturer } from "lib/public-mappings/manufacturer.ts"
+
+export default withRouteSpec({
+  ...routes["/api/v1/manufacturers/list"],
+  methods: ["GET", "POST", "OPTIONS"],
+  auth: "vercel_protection_secret",
+} as const)(async (req, res) => {
+  let filtered_manufacturers = [...req.db.manufacturers]
```

Comment:
FWIW you can avoid the `let` / mutation blocks by just having the null check inside each filter.

```
const filtered_manufacturers = manufacturers.filter(m => {
  if (req.query.integration_status == null) return true
  return manufacturer.integration === req.query.integration_status
})